media-gfx/rawtherapee: version bump to 5.8

vivo75 opened the pull request:
Signed-off-by: Francesco Riosa <vivo75@gmail.com>
Closes: https://bugs.gentoo.org/708202

a17r commented on media-gfx/rawtherapee/rawtherapee-5.8.ebuild:
```diff
@@ -0,0 +1,74 @@
+	media-libs/lensfun
+	media-libs/libcanberra[gtk3]
+	media-libs/libiptcdata
+	media-libs/libpng:0
```
missing slot op

a17r commented on media-gfx/rawtherapee/rawtherapee-5.8.ebuild:
```diff
@@ -0,0 +1,74 @@
+	media-libs/libiptcdata
+	media-libs/libpng:0
+	media-libs/tiff:0
+	sci-libs/fftw:3.0
```
missing slot op

a17r commented on media-gfx/rawtherapee/rawtherapee-5.8.ebuild:
```diff
@@ -0,0 +1,74 @@
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
```
unnecessary empty line

a17r commented on media-gfx/rawtherapee/rawtherapee-5.8.ebuild:
```diff
@@ -0,0 +1,74 @@
+	media-libs/tiff:0
+	sci-libs/fftw:3.0
+	sys-libs/zlib
+	tcmalloc? ( dev-util/google-perftools )
```
optional below unconditional deps please

a17r replied:
still not sorted, will fix after merge

a17r commented on media-gfx/rawtherapee/rawtherapee-5.8.ebuild:
```diff
@@ -0,0 +1,74 @@
+	media-libs/lcms:2
+	media-libs/lensfun
+	media-libs/libcanberra[gtk3]
+	media-libs/libiptcdata
```
media-libs/libiptcdata is a py2-only package. Is there a way to make it optional?

vivo75 replied:
I'd say it's difficult to make it optional (deeply embedded in rtexif.{cc,h}) and also still relevant for an imaging software to speak IPTC (used by Adobe extensively)
@heckflosse, @Beep6581 do you have anything to say?

Beep6581 replied:
It is not optional. Ideally we'd like to drop it entirely as libiptcdata is dead, but for the time being it's required.

a17r commented:
ugh, something went wrong here. please never merge, use rebase instead.

vivo75 commented:
@a17r yes "something" went very wrong, it wasn't a merge but having tried to keep multiple gentoo remotes or something, sorry fort the fuss, should be fixed now

vivo75 commented:
@17r I will create another pull request (after a bit of testing) for media-libs/libiptcdata-1.0.5 - https://bugs.gentoo.org/708706
That one should be accepted _before_ this one

vivo75 commented:
> only waiting for PR14626 then. ;)

Thank you for reminding me, I've totally lost email from your last comment on that PR

vivo75 commented:
> actually none of the review was addressed yet?

changes have probably been lost when I've reverted the git disaster done while trying to cope with both "rsync friendly" gentoo git repo and the actual upstream gentoo repo.
I've redone them and rebased

vivo75 commented on media-gfx/rawtherapee/rawtherapee-5.8.ebuild:
```diff
@@ -0,0 +1,73 @@
+	dev-libs/expat
+	dev-libs/libsigc++:2
+	media-libs/lcms:2
+	media-libs/lensfun:0=
```
> I don't see a subslot for lensfun.

then media-gfx/darktable should change too, I've grepped it for the lensfun DEP ;)
give me a few minutes for a new forced push
